### PR TITLE
ci: Add skipIfOnGHA util

### PR DESCRIPTION
**Pull request by seemethere**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#59748 ci: Add skipIfOnGHA util**

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D29008217](https://our.internmc.facebook.com/intern/diff/D29008217)

### CODE REVIEW

**Review by malfet — APPROVED** [review]

**Comment by seemethere** [comment]
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29008217).

**Comment by seemethere** [comment]
Closing, forward fixing with https://github.com/pytorch/kineto/pull/281

**Comment by seemethere** [comment]
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29008217).

File: torch/testing/_internal/common_utils.py [hunk 1]
```diff
@@ -391,6 +391,9 @@ def _check_module_exists(name):
 # (unlike slow tests!)
 TEST_SKIP_NOARCH = os.getenv('PYTORCH_TEST_SKIP_NOARCH', '0') == '1'
 
+# Disables tests for when on Github Actions
+ON_GHA = os.getenv("GITHUB_ACTIONS")
```
Comment (walterddr) [inline torch/testing/_internal/common_utils.py:395]:
```suggestion
ON_GHA = os.getenv('GITHUB_ACTIONS', '0') == '1'
```

Comment (seemethere) [reply torch/testing/_internal/common_utils.py]: changed

**Comment by seemethere** [comment]
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29008217).

**Review by janeyx99 — APPROVED** [review]

**Pull request closed without merging**